Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -23,8 +22,15 @@ final class SettingsAccountViewController: UIViewController {
     self.tableView.delegate = self
 
     self.tableView.register(nib: .SettingsTableViewCell)
+    self.tableView.register(nib: .SettingsAccountPickerCell)
+
     self.tableView.registerHeaderFooter(nib: .SettingsHeaderView)
 
+    let tapRecognizer = UITapGestureRecognizer(target: self,
+                                               action: #selector(tapGestureToDismissCurrencyPicker))
+    tapRecognizer.cancelsTouchesInView = false
+    self.view.addGestureRecognizer(tapRecognizer)
```

Comment:
Is there a way we could conditionally add this gesture recognizer when the picker cell is shown, and remove it when the cell is hidden?